substrate/frame/identity/src/lib.rs

```diff
@@ -280,6 +288,16 @@ pub mod pallet {
 		SubIdentityRevoked { sub: T::AccountId, main: T::AccountId, deposit: BalanceOf<T> },
 	}
 
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn integrity_test() {
+			assert!(
+				T::IdentityInformation::max_encoded_len() <= T::MaxIdentitySize::get() as usize,
```

Comment:
This should always fail, because the `max_encoded_len` of the `IdentityInforrmation` would be with all fields set at their maximum.

Reply:
Gone now with the other change.